cmd/syncer/main.go

```diff
@@ -33,11 +33,13 @@ import (
 const numThreads = 2
 
 var (
-	fromKubeconfig = flag.String("from_kubeconfig", "", "Kubeconfig file for -from cluster.")
-	fromCluster    = flag.String("from_cluster", "", "Name of the -from logical cluster.")
-	toKubeconfig   = flag.String("to_kubeconfig", "", "Kubeconfig file for -to cluster. If not set, the InCluster configuration will be used.")
-	toContext      = flag.String("to_context", "", "Context to use in the Kubeconfig file for -to cluster, instead of the current context.")
-	clusterID      = flag.String("cluster", "", "ID of the -to cluster. Resources with this ID set in the 'kcp.dev/cluster' label will be synced.")
+	fromKubeconfig         = flag.String("from_kubeconfig", "", "Kubeconfig file for -from cluster.")
+	fromCluster            = flag.String("from_cluster", "", "Name of the -from logical cluster.")
+	toKubeconfig           = flag.String("to_kubeconfig", "", "Kubeconfig file for -to cluster. If not set, the InCluster configuration will be used.")
+	toContext              = flag.String("to_context", "", "Context to use in the Kubeconfig file for -to cluster, instead of the current context.")
+	clusterID              = flag.String("cluster", "", "ID of the -to cluster. Resources with this ID set in the 'kcp.dev/cluster' label will be synced.")
+	leaseControllerEnabled = flag.Bool("lease-controller-enabled", false, "Enable lease controller (should be set to true only in pull mode).")
+	leaseDurationSeconds   = flag.Int("lease-duration-seconds", 40, "Cluster lease duration seconds.")
```

Comment:
How about having only one flag, where `--lease-duration-seconds=0` indicates there should be no lease?

Reply:
✅